app/ai-gateway/monitor-ai-llm-metrics.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "Monitor AI LLM metrics"
+content_type: reference
+layout: reference
+
+products:
+  - gateway
+
+tags:
+  - ai-gateway
+  - monitoring
+
+plugins:
+  - prometheus
+  - ai-proxy
+  - ai-proxy-advanced
+
+min_version:
+  gateway: '3.7'
+
+description: "This guide walks you through collecting AI metrics and sending them to Prometheus."
+
+related_resources:
+  - text: Kong AI Gateway
+    url: /ai-gateway/
+  - text: Kong AI Gateway plugins
+    url: /plugins/?category=ai
+  - text: Status API
+    url: /api/gateway/status/v1/
+  - text: Admin API
+    url: /api/gateway/admin-ee/3.9/
+---
+
+Kong AI Gateway calls LLM-based services according to the settings of the [AI Proxy](/plugins/ai-proxy/) and [AI Proxy Advanced](/plugins/ai-proxy-advanced/) plugins.
+You can aggregate the LLM provider responses to count the number of tokens used by the AI plugins.
+If you have defined input and output costs in the models, you can also calculate cost aggregation.
+The metrics details also expose whether the requests have been cached by {{site.base_gateway}}, saving the cost of contacting the LLM providers, which improves performance.
+
+Kong AI Gateway exposes metrics related to Kong and proxied upstream services in 
+[Prometheus](https://prometheus.io/docs/introduction/overview/) 
+exposition format, which can be scraped by a Prometheus Server.
+
+The metrics are available on both the [Admin API](/api/gateway/admin-ee/) and the 
+[Status API](/api/gateway/status/)  at the `http://<host>:<port>/metrics` endpoint. 
+Note that the URL to those APIs is specific to your
+installation. See [Accessing the metrics](#accessing-the-metrics) for more information.
+
+The [Prometheus plugin](/plugins/prometheus/) records and exposes metrics at the node level. Your Prometheus
+server will need to discover all Kong nodes via a service discovery mechanism,
+and consume data from each node's configured `/metrics` endpoint.
+
+AI metrics exported by the plugin can be graphed in Grafana using a [drop-in
+dashboard](https://grafana.com/grafana/dashboards/21162-kong-cx-ai/).
+
```

Comment:
Sounds like this isn't available on konnect - we should probably indicate that with a note somehow.

Reply:
I assumed it worked in the same way on any data plane, but I'll double-check

Reply:
Ohh I see where all this came from. It's mostly a word-for-word copy of the Prometheus plugin: https://docs.konghq.com/hub/kong-inc/prometheus/

And that's why it only talks about on-prem - that's what that doc was written for. 

I'm not 100% sure how the Konnect config works here, but I know that it does (eg see https://kongstrong.slack.com/archives/C06GB3KM155/p1736269876007219)

Reply:
So after looking around and testing a bit:
- I realized that I somehow didn't know that data planes can't use the Admin API
- The Status API iworks with the DP, I managed to make it work with the config I found [here](https://kongstrong.slack.com/archives/C04UMEJRZRP/p1680268002648999), so I'll add a note that the Status API is the only method compatible with Konnect